occrp/templates/story.html

```diff
@@ -4,27 +4,29 @@
 <div class="row">
     <div class="col-sm-12">
         <h3>{{ story.title }}</h3>
-        <p>Last update: {{ story.updated_at }}</p>
-        <p><a class="btn btn-warning" href="{{ url_for('views.index')}}"><i class="fa fa-arrow-circle-left" aria-hidden="true"></i> Return to stories list</a> </p>
-    </div>
-</div>
-
-<div class="row">
-    <div class="col-sm-12">
-        <h3>Add an event and related people, organizations, and sources to this story</h3>
-    </div>
-</div>
-
-<div class="row">
-    <div class="col-md-12">
-        {% include 'partials/event_form.html' %}
+        <p>Last update: 
+            {% if story.updated_at %}
+                <strong>{{ story.updated_at.strftime('%Y %B %d @ %H:%M') }}</strong></p>
```

Comment:
So, the events table shows dates as `%d %B %Y %H:%M` and this date and the one on the home page put the year first. We should probably be consistent.